easybuild/easyblocks/generic/intelbase.py

```diff
@@ -215,9 +215,19 @@ def prepare_step(self, requires_runtime_license=True):
         self.requires_runtime_license = self.cfg['requires_runtime_license'] and requires_runtime_license
 
         if self.requires_runtime_license:
+            license_file = self.cfg['license_file']
+            if isinstance(license_file, basestring):
+                license_specs = [license_file]
+            elif isinstance(license_file, list):
+                license_specs = license_file
+            else:
+                msg = "Unsupported type for easyconfig parameter 'license_file'! "
+                msg += "Can either be a string or a list of strings."
```

Comment:
you're not checking whether you're getting a list of strings, just whether you're getting a list...

If you want to, you could use something like:

```python
if not all(isinstance(x, basestring) for x in license_file):
```